src/modules/user/Register.ts

```diff
@@ -1,35 +1,34 @@
-import * as bcrypt from 'bcryptjs';
-import { Arg, Mutation, Query, Resolver, UseMiddleware } from "type-graphql";
-import { User } from '../../entity/User';
-import { withAuthenticatedUser } from '../middleware/withAuthenticatedUser';
-import { createAccountConfirmationUrl } from '../utils/createAccountConfirmationUrl';
-import { sendEmail } from '../utils/sendEmail';
-import { RegisterInput } from './register/RegisterInput';
+import * as bcrypt from "bcryptjs";
+import { Arg, Mutation, Query, Resolver } from "type-graphql";
+import { User } from "../../entity/User";
+// import { withAuthenticatedUser } from '../middleware/withAuthenticatedUser';, UseMiddleware
```

Comment:
why are you adding commented code to the repo? We have git history for tracking the changes